GetStream/stream-chat-swift
# [CIS-911] Regenerate Links to other reference docs in files

Your tunnel: https://stream-composer-links-fixed.loca.lt

# What this PR do:
- Fixes links to other files in reference docs so they are accessible. This was done by modifying swift-doc to add one more `../` path component to all paths due to nature of Docusaurus not handling this case.
# How to test it
- If you want to test this locally, checkout `staging` branch in `stream-chat-docusaurus-cli` repository and then run docusaurus on this branch. Links should be fixed. If you want to make sure, check out main and try to run docusaurus from the `staging` branch to see that links to other files in referenceDocs are broken.
